megamek/i18n/megamek/client/messages.properties

```diff
@@ -206,8 +214,8 @@ BoardSelectionDialog.UpdateMapSize.message=Please hit the Update Map Size button
 BoardSelectionDialog.UpdateMapSize.title=Please Update Map Size
 BoardSelectionDialog.UpdateSize=Update Size
 BoardSelectionDialog.Updating=Updating...
-BoardSelectionDialog.ViewGameBoard=View Game Board*
-BoardSelectionDialog.ViewGameBoardTooltip=<html>Shows a preview of the selected map.<br>  *Note: for generated maps the preview is only a sample<br> and will not be the same as the actual map used.</html>
+BoardSelectionDialog.ViewGameBoard=Preview Game Map... *
+BoardSelectionDialog.ViewGameBoardTooltip=Shows a preview of the game's map.<br>  * Note: For generated maps and surprise maps the preview is only a sample<br> and will not be the same as the actual map used.
```

Comment:
Recommendation: Swap the side where the BR is, to avoid having 
```
Hello
 World
```

Reply:
The "* Note" is supposed to be indented.